### DISCUSSION TITLE
Case number #00167508

### DISCUSSION THREAD

**Discussion by sinisterstumble** [vercel/community#4642] [Account Issue]
### Question

Please take a look at case numner 00167508.

Thank you

**Comment by amyegan** [comment]
Hi @markmartirosian! Thanks for sharing your case number. If you want to delete your team, you can do that from the team's settings page. Don't forget to transfer your projects and domains to your personal account before deleting the team. 

https://vercel.com/docs/accounts/create-a-team#deleting-a-team

If you are unable to delete them team from the settings page, please share a screenshot so we can see what's going wrong 🙏

**Reply by sinisterstumble** [reply]
@amyegan thank you, I have replied in the support ticket.
